outputs.tf

```diff
@@ -12,6 +12,11 @@ output "dns_name" {
   value       = try(aws_globalaccelerator_accelerator.this[0].dns_name, "")
 }
 
+output "dual_stack_dns_name" {
```

Comment:
this MSV of the AWS provider will need to be updated to [v4.61](https://github.com/hashicorp/terraform-provider-aws/blob/release/4.x/CHANGELOG.md#4610-march-30-2023) across the project